core/controllers/topic_editor.py

```diff
@@ -541,14 +544,31 @@ def get(self, topic_id: str) -> None:
         skill_creation_is_allowed = (
             role_services.ACTION_CREATE_NEW_SKILL in self.user.actions)
 
+        curriculum_admin_usernames = (
+            user_services.get_usernames_by_role('ADMIN'))
+
         self.values.update({
-            'classroom_url_fragment': classroom_url_fragment,
+            'classroom_url_fragment': (
+                None if (
```

Comment:
Don't you here put None effectively, just like I suggested below? Won't it break the learner dashboard?

Reply:
No it won't as the endpoint is used only in the creator facing part which is topic and skill dashboard.